packages/fiori/src/Wizard.ts

```diff
@@ -233,6 +234,13 @@ type StepInfo = {
 })
 
 class Wizard extends UI5Element {
+	/**
+	 * Defines how the content of the <code>ui5-wizard</code> would be visualized.
+	 * @public
+	 */
+	@property({ validator: String, defaultValue: WizardContentLayout.MultipleSteps })
+	contentLayout?: WizardContentLayout
```

Comment:
Sounds better than before, thank you for looking at it, it's not technical as before, more user-friendly.

One thing that could be confusing is that no matter the property value, we usually have multiple steps in the wizard header. The docs explains it "Defines how the content..", but at the same time you can set "SingleStep" but slot multiple WizardSteps - it's just the wording that don't match perfectly. At the ned, if none sees this as confusing, I am ok with this proposal.


*semicolon at the end are recommended WizardContentLayout**;**

Reply:
Hi, I understand where the confusion comes from, but we consider this "SingleStep" to be in the context of the "contentLayout" which describes the content of the Wizard. Therefore having multiple slotted steps doesn't necessarily mean that all steps should be displayed in the content together. Do you think that we need to enrich the documentation in order to avoid further confusion ?

Reply:
All right, it's fine to me - you can continue and merge it as it is now.